### ISSUE TITLE
[BUG] Cannot perform runtime binding on a null reference

### ISSUE THREAD

**Issue by CriticalFlaw** [CriticalFlaw/TF2HUD.Editor#231]
The following error gets logged for some users, preventing them from applying all of their customizations.

Reinstalling the editor seems to be fix the issue but should still investigate.

```
2024-07-14 00:06:41,863 | INFO | Set "font" to "0".
2024-07-14 00:06:41,865 | ERROR | Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: Cannot perform runtime binding on a null reference
   at CallSite.Target(Closure, CallSite, Object)
   at System.Dynamic.UpdateDelegates.UpdateAndExecute1[T0,TRet](CallSite site, T0 arg0)
   at HUDEditor.Classes.Utilities.Merge(Dictionary2 object1, Dictionary2 object2) in C:\Users\nik_1\OneDrive\Documents\GitHub\TF2HUD.Editor\src\TF2HUD.Editor\Classes\Utilities.cs:line 292
2024-07-14 00:06:41,887 | ERROR | Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: Cannot perform runtime binding on a null reference
   at CallSite.Target(Closure, CallSite, Object)
   at System.Dynamic.UpdateDelegates.UpdateAndExecute1[T0,TRet](CallSite site, T0 arg0)
   at HUDEditor.Classes.Utilities.Merge(Dictionary2 object1, Dictionary2 object2) in C:\Users\nik_1\OneDrive\Documents\GitHub\TF2HUD.Editor\src\TF2HUD.Editor\Classes\Utilities.cs:line 292
   at HUDEditor.Classes.Utilities.Merge(Dictionary2 object1, Dictionary2 object2) in C:\Users\nik_1\OneDrive\Documents\GitHub\TF2HUD.Editor\src\TF2HUD.Editor\Classes\Utilities.cs:line 295
2024-07-14 00:06:41,887 | ERROR | Cannot perform runtime binding on a null reference
2024-07-14 00:06:41,887 | INFO | Applying: fh_centered_item_meters
```

**Comment by CriticalFlaw** [comment]
Upon further investigation this may be related to stale AppData content that refers to customizations that are no longer exist. A robust clear cache option needs to be implemented to avoid these issues in the future see #

**Comment by CriticalFlaw** [comment]
This may have been resolved in 3.3 RC1 release - monitoring for users feedback.

**Comment by CriticalFlaw** [comment]
Fixed in the 3.3 release
